bindings/2.204/GeometryDash.bro

```diff
@@ -6843,7 +6843,7 @@ class GJGameState {
 	TodoReturn controlTweenAction(int, int, GJActionCommand);
 	TodoReturn getGameObjectPhysics(GameObject*);
 	TodoReturn processStateTriggers();
-	TodoReturn stopTweenAction(int);
+	stopTweenAction(int) = win 0x18EA30;
```

Comment:
wheres did the return go

Reply:
look it's 10pm

Reply:
actually can't figure out how to edit the code on mobile so uhhh

Reply:
Fixed it